ThreeBotPackages/threebot/contacts/html/service.js

```diff
@@ -0,0 +1,20 @@
+const headers = { headers: { 'Content-Type': 'application/json' } }
+export default ({
+  getContacts () {
+    return axios.post(`${window.config.jsApiUrl}contacts/list`)
```

Comment:
please update to use gedis client instead of axios, so you can do something like
```
localGedisClient.actors.contacts.list().then((resp) => console.log(resp.json()))
```